packages/terra-list/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,127 @@
+# Terra List - Sections with Subsections
+
+With the inclusion of sections in the list, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. It is not advised to have both sections and subsections to be collapsible.
```

Comment:
Can we expand on why it is not advised to have both sections and subsections be collapsible? Is this a UX concern, a performance concern, other?

Reply:
It is the UX guidline. Updated the comment accordingly.